modules/migration-terminology.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+[id="migration-terminology_{context}"]
+= {mtc-short} terminology
```

Comment:
regarding terminology, we also need `State`

Reply:
Will be added in separate PR for state migration.